### PR TITLE
Removed sentences on requirements about being online for other editors.

### PR DESCRIPTION
**Pull request by pelikhan**

### CODE REVIEW

**Comment by ntoll** [comment]
Hey @pelikhan, many thanks for taking the time to create this PR. :-) I'll give some context and then you'll see where I was coming from when I wrote the docs.

Through October and November of last year I ran a large number of MicroPython based workshops for teachers and developers. We wanted to find out how best to present MicroPython as well as have feedback on our Python based API for the device.

The most troubling feedback was that almost everyone hated writing Python in a browser. Problems included: 

* Users have to get the hex from the browser (and, by default, each time the browser appends a number to it so the download directory gets littered with `my_script.hex`, `my_script(1).hex`, `my_script(2).hex` etc); 
* There's no Python REPL (people love the REPL);
* Users wanted to save Python scripts to their file system;
* Users got fed up dragging and dropping scripts from the file system to the attached micro:bit.

Also, we had complaints that the browser based offering is unusable on Raspberry Pi (due to the underpowered nature of the Pi and its default browser more than anything else). This is problematic since writing Python on a Pi is an obvious next step on from writing Python on the micro:bit.

It was to these people that I was speaking in the section you highlight. I wanted to accentuate the fact that Mu isn't browser/web based and the benefits this brings in relation to the problems they had described above. I'd still like to be able to make this distinction.

I'll annotate suggested revisions to your PR in-line. Hopefully you're happy with these. As always, comments, constructive critique and ideas are most welcome.

Sound like a plan? Please update the PR and I'll merge. Thank you! :-)

File: README.rst [hunk 1]
```diff
@@ -95,12 +95,10 @@ for microcontrollers including the BBC micro:bit.
 The BBC's "blessed" solution for programming this device is web-based. However,
 we have observed that this doesn't provide the optimum experience for Python:
 
-* It requires you to be connected to the internet to get the tools you need.
```
Comment (ntoll) [inline README.rst]: * It requires you to use a web-browser as a text based code editor.

File: README.rst [hunk 2]
```diff
@@ -95,12 +95,10 @@ for microcontrollers including the BBC micro:bit.
...
 * You need to download the .hex file to flash onto the device and then drag it to the device's mount point on the filesystem. A rather clunky multi-part process.
 * It doesn't allow you to connect to the device in order to live code in Python via the REPL.
 
-The Mu editor addresses each of these problems: it is a native application so
-does not need to be connected to the internet to work. It makes it easy to
+The Mu editor addresses each of these problems: it makes it easy to
```
Comment (ntoll) [inline README.rst]: The Mu editor addresses each of these problems: it is a native application specifically designed as a text based coding environment. It makes it easy to

**Comment by pelikhan** [comment]
Your changes look good. Can you just make them?

**Comment by ntoll** [comment]
Will do, and thanks. :-)

**Pull request merged**